File: app/src/main/java/com/paradoxo/amadeus/util/Cronos.java
package com.paradoxo.amadeus.util;

import org.joda.time.DateTime;
import org.joda.time.format.DateTimeFormat;

public class Cronos {

    public static String getData(){
        return "Hoje é dia "+DateTime.now().toString(DateTimeFormat.longDate());
    }

    public static String getHora(){
        return "São "+DateTime.now().toString("HH:mm");
    }
}

File: app/src/main/java/com/paradoxo/amadeus/util/Cronos.kt
package com.paradoxo.amadeus.util

import org.joda.time.DateTime
import org.joda.time.format.DateTimeFormat

object Cronos {
    fun getData(): String = "Hoje é dia ${DateTime.now().toString(DateTimeFormat.longDate())}"

    fun getHora(): String = "São ${DateTime.now().toString("HH:mm")}"
}
